### ISSUE TITLE
ME func data: is having only magnitude _sbref a sign of a "problem"?

### ISSUE THREAD

**Issue by yarikoptic** [nipy/heudiconv#421]
Dear @pvelasco and @emdupre, I need your MExpertise. In f053c0bbbd767747db9094d56ff29fc2aa64b624  we (thanks @pvelasco !) have added code which **always** adds `_rec-` (magnitude or phase) for sbref sequences, and comment said:

            # _sbref sequences reconstructing magnitude and phase generate
            # two NIfTI files IN THE SAME SERIES, so we cannot just add
            # the suffix, if we want to be bids compliant:

I wonder if it is generally true!?  In my sample case I have only a magnitude sbref per each echo. I also see a sample openneuro dataset which seems to have just `_sbref` per each echo:
```
$> find -iname *_sbref* | grep func | grep -v _rec- | grep _echo 
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-1_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-1_sbref.nii.gz
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-2_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-2_sbref.nii.gz
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-3_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-localizer_run-01_echo-3_sbref.nii.gz
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-1_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-1_sbref.nii.gz
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-2_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-2_sbref.nii.gz
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-3_sbref.json
./ds002147/sub-7700001/ses-01/func/sub-7700001_ses-01_task-rest_run-01_echo-3_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-1_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-1_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-2_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-2_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-3_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-localizer_run-01_echo-3_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-1_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-1_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-2_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-2_sbref.nii.gz
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-3_sbref.json
./ds002147/sub-7700002/ses-01/func/sub-7700002_ses-01_task-rest_run-01_echo-3_sbref.nii.gz
```

So -- am I missing something valuable (phase... not yet sure what it could be used for), or we (in heudiconv) should permit for only a single magnitude sbref being provided, and then not bother adding `_rec-` field.

Please advise

Attn @tsalo - might also be of interest, given #387

**Comment by tsalo** [comment]
If you're not reconstructing both magnitude and phase data for the actual multiband scan, I don't think it's a problem at all. I think just getting magnitude sbrefs is probably the most common case for multiband data, whether it's also multi-echo or not, although I guess some sequences might reconstruct the complex data for the sbrefs and not the main scans by default.

As a minor note, ultimately I think the best entity for the magnitude/phase keyword will be `part`, rather than `rec`. Granted, `part` isn't actually in the specification (yet), but it is proposed for separating magnitude and phase data in BEP-001 and BEP-004.

**Comment by pvelasco** [comment]
You only get phase for the sbref series if you selected to also reconstruct the phase in the corresponding multiband run, whether is ME or not. Most people don't (they save only the magnitude), so the sbref series only have magnitude images.
I didn't realize that my commit adds the `_rec-magnitude` field to all ME sbref series, whether there were phase images or not.
I can work on a fix for that, so that it only adds the `_rec-magnitude` if there are also phase images.

**Comment by yarikoptic** [comment]
fixed by 0eb5fbd9f40bad75ad5f081598dd66fe3aa7b0b3 included within #387
